airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -707,20 +707,6 @@ def read_records(self, sync_mode: SyncMode, stream_slice: Optional[Mapping[str,
             items_next_pages = super().read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream_slice, **kwargs)
         yield from chain(items, items_next_pages)
 
-    def stream_slices(
```

Comment:
@darynaishchenko can you confirm this method was only added for logging purposes as part of [this PR](https://github.com/airbytehq/airbyte/pull/33038/files#diff-bc939ec7f5dc075809864e54aca658012af002c91181af90b6b058de17bc4205R703-R715)?

This is creating a lot of log entries (one per parent record), so I suggest we revert this

Reply:
@girarda yes, that is only for logs purpose